### PR TITLE
Style and implementation of profile menu

### PR DESCRIPTION
**Pull request by jules-n**
Should be included to header.
Functionality:
 - opening profile page
 - redirect to boards page
 - logging out

### CODE REVIEW

**Review by ChernykhVyacheslav — APPROVED** [review]

File: src/components/Header/ProfileMenu/ProfileMenu.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
+import { Link } from 'react-router-dom';
+import React, { Component } from 'react';
+import $ from 'jquery';
+import './profileMenu.css';
+import ProfilePage from '../../ProfilePage/ProfilePage';
+
+class ProfileMenu extends Component{
```
Comment (achehrynets) [inline src/components/Header/ProfileMenu/ProfileMenu.js:7]: Fix code style:
`Component{`
One space should be between `Component` and `{`

File: src/components/Header/ProfileMenu/ProfileMenu.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    state = {
+        seen: false
+    };
+
+    render (){
```
Comment (achehrynets) [inline src/components/Header/ProfileMenu/ProfileMenu.js:12]: Fix code style:
One space should be between `)` and `{`

File: src/components/Header/ProfileMenu/ProfileMenu.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    const menu_window = () =>{
```
Comment (achehrynets) [inline src/components/Header/ProfileMenu/ProfileMenu.js:13]: Fix code style:
One space should be between ) and {

File: src/components/Header/ProfileMenu/ProfileMenu.js [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+        $('.user_submenu').toggle();
+        $('.message_box').hide();
+    };
+
+    const clear_token = () =>{
```
Comment (achehrynets) [inline src/components/Header/ProfileMenu/ProfileMenu.js:18]: Fix code style:
One space should be between ) and {

File: src/components/Header/ProfileMenu/ProfileMenu.js [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+        sessionStorage.clear();
+    };
+
+    const profile_window = () =>{
```
Comment (achehrynets) [inline src/components/Header/ProfileMenu/ProfileMenu.js:22]: Fix code style:
One space should be between ) and {

File: src/components/Header/ProfileMenu/profileMenu.css [hunk 6]
```diff
@@ -0,0 +1,48 @@
+.user_submenu{
```
Comment (achehrynets) [inline src/components/Header/ProfileMenu/profileMenu.css:1]: Fix code style:
One space should be between `user_submenu` and `{`

File: src/components/Header/ProfileMenu/profileMenu.css [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+    background: #e9e9e923;
+    border-right: 1px solid rgba(56, 109, 216, 0.993);
+    border-bottom: 1px solid rgba(56, 109, 216, 0.993);
+    border-left: 1px solid rgba(56, 109, 216, 0.993); 
+    overflow: hidden; 
+    display: none;
+    width: 9%;
+}
+
+.user_icon{
```
Comment (achehrynets) [inline src/components/Header/ProfileMenu/profileMenu.css:14]: Fix code style

File: src/components/Header/ProfileMenu/profileMenu.css [hunk 8]
```diff
@@ -0,0 +1,48 @@
...
+    display: flex;
+    align-self: center;
+}
+
+.user_submenu ul{
```
Comment (achehrynets) [inline src/components/Header/ProfileMenu/profileMenu.css:19]: Fix code style:

File: src/components/Header/ProfileMenu/profileMenu.css [hunk 9]
```diff
@@ -0,0 +1,48 @@
...
+    list-style-type: none;
+    min-width: 150px;
+    padding: 0;
+    margin: 0;
+}
+
+.user_submenu ul li{
```
Comment (achehrynets) [inline src/components/Header/ProfileMenu/profileMenu.css:26]: Fix code style:

File: src/components/Header/ProfileMenu/profileMenu.css [hunk 10]
```diff
@@ -0,0 +1,48 @@
...
+    background: white;
+    color: black;
+    opacity: 1;
+    padding: 5%;
+    display: flex;
+    justify-content: space-between;
+    border-bottom: 1px solid #e4e4e4;
+}
+
+.user_submenu ul li:hover{
```
Comment (achehrynets) [inline src/components/Header/ProfileMenu/profileMenu.css:37]: Fix code style:

File: src/components/Header/ProfileMenu/profileMenu.css [hunk 11]
```diff
@@ -0,0 +1,48 @@
...
+    background: rgb(115, 161, 253);
+}
+
+.user_submenu ul li i{
```
Comment (achehrynets) [inline src/components/Header/ProfileMenu/profileMenu.css:41]: Fix code style

File: src/components/Header/ProfileMenu/profileMenu.css [hunk 12]
```diff
@@ -0,0 +1,48 @@
...
+    padding-right: 10px;
+    font-size: 16px;
+}
+
+li a{
```
Comment (achehrynets) [inline src/components/Header/ProfileMenu/profileMenu.css:46]: Fix code style

File: src/components/Header/ProfileMenu/ProfileMenu.js [hunk 13]
```diff
@@ -0,0 +1,46 @@
...
+                                <ul>
+                                    <li id="profile"><a onClick = {profile_window}><span>Profile</span></a></li>
+                                    <li id="boards"><Link to="/boards"><span>Boards</span></Link></li>
+                                    <li id="logout"><Link onClick = {clear_token} to="/"><span>Log out</span></Link></li>
+                                </ul>
+                            </div>
+                        </div>
+            {this.state.seen ? <ProfilePage  onClick={profile_window}/> : null}
+        </div>
+    )
```
Comment (achehrynets) [inline src/components/Header/ProfileMenu/ProfileMenu.js:42]: Fix code formatting, it (`)`) cannot be on the same space level as the end of method block (`}`)

**Review by achehrynets — CHANGES_REQUESTED** [review]
In general looks good, but please fix code style issues.

**Review by achehrynets — APPROVED** [review]
Looks good!

**Pull request merged**